charts/artifact-hub/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: artifact-hub
 description: Artifact Hub is a web-based application that enables finding, installing, and publishing Kubernetes packages.
 type: application
-version: 1.9.0
+version: 1.9.0-1
```

Comment:
Thanks for the quick reply! 

Yes, of course. I adapted the version number. 👍 

I was not 100% sure regarding the version scheme of the helm chart. Maybe this could be added to the CONTRIBUTING.md 🤔